src/App.js

```diff
@@ -1,38 +1,29 @@
 import logo from './logo.svg';
 import './App.css';
-
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 // function getTitle(title){  return title;}
 
 const title = 'React';
 const list=[
   {id:1,title:'React',url:'https://react.js.org/',objectID:0,num_comments:3,points:4},
   {id:2,title:'Rdux',url:'https://redux.js.org/',objectID:1,num_comments:4,points:5}
 ];
+ 
+
 
 function listItem(item)
-{return 
-<li>{item.objectID} {item.title}  <a href={item.url}>{item.url}</a> {item.points}</li>
+{return  
+ <li>{item.objectID} {item.title}  <a href={item.url}>{item.url}</a> {item.points}</li>  
 }
 
 function App() {
   return (
     <div  >
-    {/* <h1>Hello {title}</h1> */}
-    {/* <h1>Hello {getTitle('World')}</h1>  */}
-
-    <h1></h1>
+   <TodoList/>
 
-    <label htmlFor="search">Search: </label>
-    <input id="search" type="text"/>
-{/* render the list */}
-<hr/>
-<ul>
-{list.map( 
-  function(item){return <li>{item.id} | {item.title} | {item.objectID} <a href={item.url}>| {item.url}</a> | {
-    item.points}</li> }
-  
-)}
-</ul>
+   <hr/>
+   <AddTodoForm/>
```

Comment:
Great job rendering the TodoList and AddTodoForm components! 💯 

I would suggest to comment out the extra code that is not being used if you would like to keep it for future reference. For example from line 7 to line 18 can be inside of a comment.